cmd/exec.go

```diff
@@ -130,9 +129,7 @@ func execRun(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if verbose {
-		fmt.Fprintf(os.Stdout, "info: With environment %s\n", strings.Join(env, ","))
-	}
+	slog.Debug(fmt.Sprintf("info: With environment %s\n", strings.Join(env, ",")))
```

Comment:
is there any world where this is considered a breaking change? It looks like the comment in the help output for verbose before was wrong since partially we output to STDERR (only this line apparently was to STDOUT)

Do we need to keep that consistency for any reason?